src/diff-snapshot.js

```diff
@@ -51,6 +52,50 @@ const fillSizeDifference = (width, height) => (image) => {
   return image;
 };
 /* eslint-enabled */
+const defaultPixelmatchDiffConfig = {
+  threshold: 0.01,
+};
+
+const defaultSSIMDiffConfig = { ssim: 'bezkrovny' };
```

Comment:
Can you add a comment above this code about what this means and where we can find out more about it? For subsequent contributors to the code to know!

Reply:
This is done along with updates to the README providing options, best practices, and links to other sources.